app/views/questions/new.html.erb

```diff
@@ -5,23 +5,74 @@
         <div class="box">
           <h2 class="title-no-margin center">New Question</h2>
           <br>
-            <%= form_for @question do |f| %>
-              <div class="form-group">
```

Comment:
Why were all the `.form-group`s removed? http://getbootstrap.com/css/#forms-example